CHANGELOG.md

```diff
@@ -7,7 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org).
 
 ## [Zebra 1.4.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.4.0) - TODO: DATE
 
-Zebra's mining RPCs are now available in release builds. TODO: rest of intro
+Zebra's mining RPCs are now available in release builds. Our Docker images are significantly smaller,
+because the Zcash verification parameters are now built into the `zebrad` binary.
```

Comment:
This is the introduction, so I'd like a one-line summary. I'm happy to just say "smaller Zcash verification parameters". (Which are different from the proving parameters.)

```suggestion
because the smaller Zcash verification parameters are now built into the `zebrad` binary.
```